### PR TITLE
Issue33

### PR DESCRIPTION
**Pull request by propardhu**

### CODE REVIEW

**Comment by propardhu** [comment]
Annotation 2020-08-06 195831

**Comment by propardhu** [comment]
Annotation 2020-08-06 195951

File: src/components/footer/Footer.vue [hunk 1]
```diff
@@ -26,8 +26,15 @@
                               </h5>
                             </v-card-title>
 
-                            <v-card-text v-for="(listItem, j) in item.list" :key="j" class="ml-5 pa-1">
-                              <v-btn text color="white" small :to="listItem.src"> <v-icon left>mdi-circle-small </v-icon> {{ listItem.title }} </v-btn>
+                            <v-card-text
+                              v-for="(listItem, j) in item.list"
```
Comment (haigopi) [inline src/components/footer/Footer.vue:30]: Breaking the format. Set your line width 180.

File: src/components/home/toolbar/ToolBar.vue [hunk 2]
```diff
@@ -2,11 +2,16 @@
   <v-container fluid class="pa-0 ma-0">
     <v-app-bar :style="tbStyle" text-center align="center" class="pa-0" elevate-on-scroll fixed>
       <v-app-bar-nav-icon>
-        <v-img :src="image" class="ml-8" width="48px"></v-img>
+        <router-link to="/"><v-img :src="image" class="ml-8" width="48px"></v-img></router-link>
       </v-app-bar-nav-icon>
 
-      <v-toolbar-title class="ml-2 text-h6 white--text font-weight-bold gradient-text-logo">VantaShala</v-toolbar-title>
+      <v-toolbar-title class="ml-2 pt-2 text-h6 white--text font-weight-bold gradient-text-logo"><router-link to="/">VantaShala</router-link></v-toolbar-title>
```
Comment (haigopi) [inline src/components/home/toolbar/ToolBar.vue:8]: why top 2?

**Comment by haigopi** [comment]
Unable to Merge.

**Pull request closed without merging**